src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -17,21 +28,140 @@ const bgForType = (type: CardType) => {
 const Match = ({ match, label, bg }: { match: number; label: string; bg: string }) => {
   return (
     <div
-      className={`${bg} flex flex-row shrink-0 sm:flex-col rounded-[16px] sm:rounded-[40px] sm:min-h-[80px] w-[132px] sm:w-[80px] h-[32px] text-white justify-center text-center items-center`}
+      className={`${bg} flex flex-row shrink-0 sm:flex-col rounded-lg sm:rounded-[40px] sm:min-h-[80px] w-[132px] sm:w-[80px] h-[32px] text-white justify-center text-center items-center`}
     >
       <span className="mr-3 sm:mr-0 font-semibold text-[22px] sm:text-[24px]">{`${match}%`}</span>
-      <span className="flex justify-center text-[12px] font-bold">{label}</span>
+      <span className="flex justify-center text-[12px]/[16px] font-arial font-bold">{label}</span>
```

Comment:
Just a note for discussion:
This seemed really weird `text-[12px]/[16px]` and I did need to check what it means.
Maybe it is more readable to use the `leading-`? So in this case it would be `leading-[16px]`

Reply:
You are right, it does look kinda confusing without reading the documentation. I'll use the `leading` class as it's easier to undestand.

Reply:
You are right, it does look kinda confusing without reading the documentation. I'll use the leading class as it's easier to undestand.